assets/js/vue/components/search/ResultSet.vue

```diff
@@ -3,7 +3,7 @@
         <section class="section-content pt-3">
             <div class="row d-flex flex-row justify-content-center">
                 <h5>
-                    Found {{ results.count }} results for: "{{ results.formValues.query }}"
+                    Found #{{ results.count }} results
```

Comment:
```suggestion
                    Found {{ results.count }} results
```

Reply:
In the ticket, I'm pretty sure the '#' character was meant to represent the number returned.